@mansnils can you please check below error 

`tensorflow/lite/micro/kernels/activations.cc:40:48: error: implicit conversion increases floating-point precision: 'float' to 'double' [-Werror,-Wdouble-promotion]
  double real_multiplier = input->params.scale / output->params.scale;
         ~~~~~~~~~~~~~~~   ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~/tensorflow/lite/micro/kernels/activations.cc:40:48: error: implicit conversion increases floating-point precision: 'float' to 'double' [-Werror,-Wdouble-promotion]
  double real_multiplier = input->params.scale / output->params.scale;
         ~~~~~~~~~~~~~~~   ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~/tensorflow/lite/micro/kernels/activations.cc:126:7: note: in instantiation of function template specialization 'tflite::ops::micro::activations::ReluQuantized<signed char>' requested here
      ReluQuantized<int8_t>(input, output, GetTensorData<int8_t>(input),
      ^/tensorflow/lite/micro/kernels/activations.cc:40:48: error: implicit conversion increases floating-point precision: 'float' to 'double' [-Werror,-Wdouble-promotion]
  double real_multiplier = input->params.scale / output->params.scale;
         ~~~~~~~~~~~~~~~   ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
/tensorflow/lite/micro/kernels/activations.cc:131:7: note: in instantiation of function template specialization 'tflite::ops::micro::activations::ReluQuantized<unsigned char>' requested here
      ReluQuantized<uint8_t>(input, output, GetTensorData<uint8_t>(input),
      ^`

--- a/tensorflow/lite/micro/kernels/activations.cc
+++ b/tensorflow/lite/micro/kernels/activations.cc
@@ -37,7 +37,7 @@ inline void ReluQuantized(const TfLiteTensor* input, TfLiteTensor* output,
   ReluParams params;
   float act_min = 0.0;
   float act_max = std::numeric_limits<float>::infinity();
-  double real_multiplier = input->params.scale / output->params.scale;
+  double real_multiplier = static_cast<double>(input->params.scale / output->params.scale);
 
   const RuntimeShape input_shape = GetTensorShape(input);
   const RuntimeShape output_shape = GetTensorShape(output);

